### DISCUSSION TITLE
Standards for Audio, Video and 3D Lens Publications / Collects

### DISCUSSION THREAD

**Discussion by mvanhalen** [aave/lens-api-examples#128] [General]
What are some Lens Specific standards which popular Lens Apps use to make sure content is rendered correctly on all apps?

Got some feedback from Paul Burke about the audio / video specifics

"I noticed that a few of the apps add custom attributes for type with the value being 'video' or 'audio'. Also an author field is used for audio files. I add both of these with Focalize."

From Sasi 
"add to the above points, "animation_url" is required to render the media content on marketplaces"

What more are there? And for 3D? What should be added or taken in consideration?

**Comment by sasicodes** [comment]
All apps should follow this standard - https://docs.lens.xyz/docs/metadata-standards

In addition to this, lenster used the custom attribute `author` for rendering author names. I don't know why we need `audio` and `video` in custom attributes. (maybe those apps are handling some logic with that but we have `mainContentFocus`)

NFT marketplaces use `animation_url` to render audio/video.

Feel free to add your ideas 🙇🏻  - https://github.com/lens-protocol/api-examples/discussions/101
